### PR TITLE
Add support for Kwenta.io

### PR DESCRIPTION
**Pull request by 0xDaedalus**
### To Test
- [ ] Navigate to https://kwenta.io/exchange/
- [ ] Connect your wallet
- [ ] Attempt a swap
- [ ] 🎉

### CODE REVIEW

**Review by jagodarybacka — DISMISSED** [review]
It works! ✨

**Review by jagodarybacka — APPROVED** [review]

File: src/window-provider.ts [hunk 1]
```diff
@@ -125,6 +125,7 @@ Object.defineProperty(window, "ethereum", {
           // The same is true for `galaxy.eco`
           if (
             (window.location.href.includes("app.uniswap.org") ||
+              window.location.href.includes("kwenta.io") ||
```
Comment (greg-nagy) [inline src/window-provider.ts:128]: Why did you need this? kwenta is using onboard and this is not required for onboard
